### PR TITLE
Remove 4 HHS Domains

### PR DESCRIPTION
**Pull request by KeithBonesJr**
#  #

## 🗣 Description ##

This pull request removes the following HHS domains from current-federal-non-dotgov.csv:

- fdastudios.com
- fdastudios.net
- fdastudios.org
- fdastudios.us

## 💭 Motivation and context ##

HHS has requested that we remove FDAstudios.com so that it no longer shows up in their Trustymail and HTTPS reports. I verified it does not belong to the agency and is not managed exclusively on their behalf.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [X] All new and existing tests pass.

### CODE REVIEW

File: current-federal-non-dotgov.csv [hunk 1]
```diff
@@ -399,7 +399,6 @@ fakecheck.us,Federal Agency - Executive,United States Postal Service,,Washington
 faqusps.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 fastmaildelivery.net,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 fastmaildelivery.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
-fdastudios.com,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.net,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.org,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.us,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
```
Comment (dav3r) [inline current-federal-non-dotgov.csv]: What about these other three similar domains?
* fdastudios.net
* fdastudios.org
* fdastudios.us

Are those still claimed by HHS?

Comment (KeithBonesJr) [reply current-federal-non-dotgov.csv]: Hey @dav3r  I was thinking the same thing as well. They don't appear to be active domains. The stakeholder only requested the removal of the .com but I did a check and those domains appear to not belong to the agency either.

**Comment by KeithBonesJr** [comment]
@dav3r I went ahead and removed the additional 3 domains.

File: current-federal-non-dotgov.csv (hunk 1 above)
Comment (dav3r) [reply current-federal-non-dotgov.csv]: Excellent, I saw https://github.com/cisagov/scan-target-data/pull/104/commits/b821fe5abf54c44be4d3a47b2f972da2fa1d5e60 and it looks good. 👍

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by jsf9k — APPROVED** [review]

**Pull request merged**